netcore/utility/SimpleSecurityManager.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package javaobject;
+
+import org.apache.geode.security.AuthenticationFailedException;
+import org.apache.geode.security.SecurityManager;
+
+import java.util.Properties;
+
+import javaobject.UserPasswordAuthInit;
+import javaobject.UsernamePrincipal;
+
+/**
+ * This Security manager only Authenticates - and allows any operations.
+ */
+public class SimpleSecurityManager implements SecurityManager {
```

Comment:
Don't we have a version of this already in the existing repo? We shouldn't need more than one fake `SecurityManager` for testing.

Reply:
Copied this from the main repo, because the earlier procedure was clumsy:
  * Build NC
  * Build installer
  * Build examples (this builds the .jar file)
  * set environment variable to root of the example build, so that `startserver.sh/startserver.ps1` can set CLASSPATH in `gfsh` command
  
I'm open to suggestions for improvement, but not a fan of tying .net core build to the main build, especially not (ugh!) the examples.

Reply:
I am cool with that. Of all the issues raised in this PR I think we only need to address the namespace.

Reply:
https://issues.apache.org/jira/browse/GEODE-9422 filed to remove duplicate Java code.